mkdocs/themes/readthedocs/breadcrumbs.html

```diff
@@ -13,11 +13,13 @@
     {% if page %}<li>{{ page.title }}</li>{% endif %}
     <li class="wy-breadcrumbs-aside">
       {%- block repo %}
-      {% if repo_url %}
-        {% if repo_name == 'GitHub' %}
-          <a href="{{ repo_url }}" class="icon icon-github"> Edit on GitHub</a>
+      {% if edit_url %}
```

Comment:
This should be `{% if page and page.edit_url %}`. As mentioned in another comment, `edit_url` is page specific. I'm okay with no "edit " link when not on a (markdown generated) "page" as there is nothing to edit anyway.